types/node/v6/index.d.ts

```diff
@@ -358,10 +358,19 @@ declare namespace NodeJS {
         isTTY?: true;
     }
 
+    export interface WriteStream extends Socket {
+        columns: number;
```

Comment:
All these properties should be optional. Because according to the https://nodejs.org/api/process.html#process_process_stdin 
> It is a net.Socket (which is a Duplex stream) unless fd 1 refers to a file, in which case it is a Writable stream.